lib/galaxy/managers/users.py

```diff
@@ -544,10 +542,9 @@ def send_reset_email(self, trans, payload, **kwd):
                     trans.request.host,
```

Comment:
Should `host` 2 lines above be replaced by `trans.app.config.hostname` and the `__get_host()` method removed?

Reply:
But where is `config.hostname` set?

Reply:
In my suggestion above :)
It should probably be moved out of `_set_email_from()` and into `_process_config()` line 1051, that should have been a better suggestion on my part.